wls-broadcast-service/src/test/java/de/muenchen/oss/wahllokalsystem/broadcastservice/security/BroadcastSecurityTest.java

```diff
@@ -6,7 +6,8 @@
 import de.muenchen.oss.wahllokalsystem.broadcastservice.TestConstants;
 import de.muenchen.oss.wahllokalsystem.broadcastservice.rest.BroadcastMessageDTO;
 import de.muenchen.oss.wahllokalsystem.broadcastservice.service.BroadcastService;
-import de.muenchen.oss.wahllokalsystem.broadcastservice.utils.BroadcastSecurityUtils;
+import de.muenchen.oss.wahllokalsystem.broadcastservice.utils.Authorities;
+import de.muenchen.oss.wahllokalsystem.wls.common.testing.SecurityUtils;
```

Comment:
Bitte auch an den Stellen verwenden wo `SecurityContextHolder.getContext().setAuthentication` aktuell verwendet wird.

Reply:
Done in a5ff9ede113df2d6a85a6d4b02c2d04843f109a3